src/main/java/com/example/online_shop_api/Controller/ProductHelpersController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.online_shop_api.Controller;
+
+import com.example.online_shop_api.Entity.ProductHelpers.Brand;
+import com.example.online_shop_api.Entity.ProductHelpers.Color;
+import com.example.online_shop_api.Service.Products.ProductHelpersService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/brand")
+public class ProductHelpersController {
+  private final ProductHelpersService productHelpersService;
+
+  @GetMapping("/getAllBrand")
```

Comment:
Since the @RequestMapping is "/brand", I think it would be a good idea to change this endpoint to /get-all? 
+we made team agreement that we will use "-" as a separator for the endpoints, not camelCase.